test/intl402/PluralRules/prototype/select/notation.js

```diff
@@ -0,0 +1,20 @@
+// Copyright 2025 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-pluralruleselect
+description: Checks that the notation option is used appropriately in the select method.
+info: |
+  PluralRuleSelect (
+    _locale_: a language tag,
+    _type_: *"cardinal"* or *"ordinal"*,
+    _notation_: a String,
+    _s_: a decimal String,
+  ): *"zero"*, *"one"*, *"two"*, *"few"*, *"many"*, or *"other"*
+    ...
+    The returned String characterizes the plural category of _s_ according to _locale_, _type_ and _notation_.
+    ...
+---*/
+
+assert.sameValue(new Intl.PluralRules('fr-FR', { notation: 'compact' }).select(1.00000020e6), 'one', 'compact notation');
```

Comment:
Shouldn't this be `'many'`, given that the value formats the same as `1c6`?

Reply:
Actually, the [CLDR plural rule](https://github.com/unicode-org/cldr/blob/faa2818490a34a2288f307720ab0abdab8c91ff7/common/supplemental/plurals.xml#L123-L127) examples for `many` even include `1.0000001c6`.

Reply:
good point, I guess this applies to all related romance languages with a similar definition of `"many"` so the original example must've been unsound. Replacing it with a different language without that category like Slovene. Does that sound like a good idea?